### ISSUE TITLE
Avoid submitting Youtube video that are not categorized as music

### ISSUE THREAD

**Issue by h3xstream** [web-scrobbler/web-scrobbler#1590]
This is a suggestion to avoid many of the false positive on Youtube.

Youtube has a category list that users need to choose from when uploading a video.

Video clip and songs are _always_ categories as "Music" ( Full list of categories https://gist.github.com/dgp/1b24bf2961521bd75d6c)

With either a blacklist or a whitelist of categories, this should reduce a lot of unnecessary submissions.

**Comment by alexesprit** [comment]
This is alreadly implemented, isn't it?
![image](https://user-images.githubusercontent.com/1119267/36943393-c0f244c4-1f99-11e8-8406-ea0b9380b5c3.png)

**Comment by h3xstream** [comment]
Thanks

**Comment by OnyxxOr** [comment]
Hello guys,
Firstly, thanks so much for this plugin, I use it frequently!
Then, I apologize if I am breaking guideline by adding further to this issue. I just figured to add a thought, as I see the number of submissions is becoming a big problem with the limited keys you have. I did not know of this setting until today, and was about to request something similar (To avoid annoying scrobbling of adverts really).

Is it possible to announce this setting as a one-time popup for users (who can enable to avoid scrobbling non-music). Perhaps this helps with the number of scrobbles, at least a little? I'm sure most users of your brilliant extension do know about this setting (Buys some time).

Best Regards!!
Dylan
